discovery-infra/test_infra/helper_classes/entity_operations.py

```diff
@@ -0,0 +1,61 @@
+from abc import ABC, abstractmethod
+
+from test_infra.helper_classes.config import BaseEntityConfig, BaseClusterConfig, BaseInfraEnvConfig
+
+
+class _EntityOperations(ABC):
+    @property
+    @abstractmethod
+    def name(self) -> str:
+        raise NotImplemented
+
+    @property
+    @abstractmethod
+    def terraform_params(self) -> dict:
+        raise NotImplemented
+
+    @property
+    @abstractmethod
+    def name_suffix(self) -> str:
+        raise NotImplemented
```

Comment:
`pass` is the same as `raise NotImplemented` on `@abstractmethod`

Reply:
I know.. missed that :)